template/src/badges.ts

```diff
@@ -0,0 +1,60 @@
+function makeIcon(svg: string): string {
+    return `data:image/svg+xml;base64,${btoa(
+        svg.replace(/^\s+/g, '').replace('\n', '')
+    )}`
+}
+
+/**
+ * Creates an icon with the material design 'information-outline' style.
+ *
+ * @param color The color of the lines.
+ */
+function makeInfoIcon(color: string): string {
+    return makeIcon(`
+        <svg xmlns='http://www.w3.org/2000/svg' style="width:24px;height:24px" viewBox="0 0 24 24" fill="#${color}">
```

Comment:
`fill` can take color names, hsl, rgb, etc. values, so I would not instinctively assume that `color` is a non-prefixed hex color value.

```suggestion
        <svg xmlns='http://www.w3.org/2000/svg' style="width:24px;height:24px" viewBox="0 0 24 24" fill="${color}">
```